uswds/uswds-site
# USWDS-Site: Update snyk ignore (October)

# Summary

Ignores two snyk vulnerabilities causing build and `npx snyk test` errors.

## Related issue

Closes #2321

## Related PR

[Update snyk ignore (September)](https://github.com/uswds/uswds-site/pull/2287)

## Problem statement

Build and `npx snyk test` and test builds were failing. 

The failures are the same as last month's Snyk Update
- SNYK-JS-UNSETVALUE-2400660
- SNYK-JS-ANSIREGEX-1583908

## Solution

Re-add the snyk ID's to the snyk ignore

## Testing and review

1. Run `npx run test` and check for errors
2. Check to see if the automated tests on this branch pass.
